### DISCUSSION TITLE
Probably typo error in source code

### DISCUSSION THREAD

**Discussion by llaville** [CycloneDX/cyclonedx-php-composer#365] [General]
Hello,

By running the `demo` projects and checking architecture (like it !), I've found probably a typo error in source code.

I've noticed that you used the term `compoer` : do you mean `composer` instead ?

Checking source code, to find where to fix if condition is true : `grep compoer ./src composer.json -R`

Preview on tag 4.0.1

```
./src/MakeBom/Command.php:        $io->writeError('<info>generate base SBOM from compoers\'s evidences...</info>', verbosity: IOInterface::VERBOSE);
composer.json:        "test:composer-require-checker": "Test compoer requirements with composer-require-checker.",
composer.json:        "test:composer-unused": "Test compoer requirements with composer-unused.",
```

**Comment by jkowalleck** [comment]
thank you for the report.
are you interested in fixing this with a pull request? If so, please reference #367 in the pullrequest.
